tagged-localbuild.xml

```diff
@@ -27,25 +27,25 @@
   <project name="device-sony-tama" path="device/sony/tama" remote="sony" revision="6734a393f25b131f1d52cd3cbbf58f71b29f154d" upstream="q-mr1" groups="device"/>
   <project name="device-sony-voyager" path="device/sony/voyager" remote="sony" revision="674b0de906a8f9891b76e77f9b1ffc445b73e96f" upstream="q-mr1" groups="device"/>
   <project name="device-sony-yoshino" path="device/sony/yoshino" remote="sony" revision="a00e91951dcd17af046db4e4a61b63582306cd17" upstream="q-mr1" groups="device"/>
   <project name="device/google_car" revision="e9080f3f999eeb872ff73aa0f8d39a48f07c7e11" upstream="refs/tags/android-10.0.0_r41" groups="pdk"/>
   <project name="device/sample" revision="d64a03ec745d8e4085ae65a868f1e8d66b2d9c4c" upstream="refs/tags/android-10.0.0_r41" groups="pdk"/>
   <project name="exfatprogs" path="vendor/external/exfatprogs" remote="exfatprogs" revision="d9fbffd53e2e1ec89584b8c6931d8984e9894f4b" upstream="refs/tags/1.0.4" groups="device"/>
-  <project name="hardware-qcom-display" path="vendor/qcom/opensource/display" remote="sony" revision="e5b6ff2cd530846e8ea1c8f776d463c6c0428724" upstream="aosp/LA.UM.8.1.r1" groups="device"/>
+  <project name="hardware-qcom-display" path="vendor/qcom/opensource/display" remote="sony" revision="4a10496eef90811bf962c0fb68bfedb041ee77e3" upstream="aosp/LA.UM.8.1.r1" groups="device"/>
   <project name="hardware-qcom-gps" path="vendor/qcom/opensource/gps" remote="sony" revision="d7bbd33c27d17a4cafafadd9ac6f806d095a53d5" upstream="aosp/LA.UM.8.1.r1" groups="device"/>
   <project name="hardware-qcom-media" path="vendor/qcom/opensource/media/sdm660-libion" remote="sony" revision="c4da76d85056261a7cf6c78dc377d40b602f06ea" upstream="aosp/LA.UM.8.2.1.r1" groups="device"/>
   <project name="hardware-qcom-media-sm8150" path="vendor/qcom/opensource/media/sm8150" remote="sony" revision="7c57b64bbaa26f5308177e5399956734a3300ff3" upstream="aosp/LA.UM.8.1.r1" groups="device"/>
   <project name="hardware-qcom-wlan" path="vendor/qcom/opensource/wlan" remote="sony" revision="6351125ec900f421276213a2a0d336cfc6268042" upstream="master" groups="device"/>
   <project name="json-c" path="vendor/oss/json-c" remote="sony" revision="3b0d70d918bc57153721d45645cb953a493ebe19" upstream="q-mr1" groups="device"/>
   <project name="kernel-defconfig" path="kernel/sony/msm-4.14/kernel/arch/arm64/configs/sony" remote="sony" revision="95c9437cb5a6bcfc1d877fdd5c59d0322b2aebbd" upstream="aosp/LA.UM.7.1.r1" groups="device"/>
-  <project name="kernel-sony-msm-4.14-common" path="kernel/sony/msm-4.14/common-kernel" remote="sony" revision="7e4350c2c566e7df37e3ed421d57a07a9396c2af" upstream="aosp/LA.UM.7.1.r1" groups="device" clone-depth="1"/>
+  <project name="kernel-sony-msm-4.14-common" path="kernel/sony/msm-4.14/common-kernel" remote="sony" revision="9315ff02ddbc7ee7edf6486a089b50ea8efdefd0" upstream="aosp/LA.UM.7.1.r1" groups="device"/>
```

Comment:
`clone-depth` is still there in sony's repos. Was it a manual change that sneaked in?:) https://github.com/sonyxperiadev/local_manifests/blob/android-10.0.0_r41/LA.UM.7.1.r1.xml#L5
```suggestion
  <project name="kernel-sony-msm-4.14-common" path="kernel/sony/msm-4.14/common-kernel" remote="sony" revision="9315ff02ddbc7ee7edf6486a089b50ea8efdefd0" upstream="aosp/LA.UM.7.1.r1" groups="device" clone-depth="1"/>
```

Reply:
ah yeah, done